oidc_provider/views.py

```diff
@@ -240,6 +241,16 @@ def userinfo(request, *args, **kwargs):
 
     Return a dictionary.
     """
+
+    def set_headers(response):
+        response['Cache-Control'] = 'no-store'
+        response['Pragma'] = 'no-cache'
+        cors_allow_any(request, response)
+        return response
+
+    if request.method == 'OPTIONS':
+        return set_headers(HttpResponse())
```

Comment:
Why is set_headers defined as a separate function here?
I think this should be simply:
```python
if request.method == 'OPTIONS':
    response = HttpResponse()
    response['Cache-Control'] = 'no-store'
    ...
    return response
```